legacy/src/main/java/org/opensearch/sql/legacy/executor/format/PrettyFormatRestExecutor.java

```diff
@@ -90,20 +98,49 @@ public String execute(Client client, Map<String, String> params, QueryAction que
   private Protocol buildProtocolForDefaultQuery(Client client, DefaultQueryAction queryAction)
```

Comment:
When is this method used?

Reply:
This is called while formatting the search response received by executing any query action plan.